Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Brann Bronzebeard Changes

Moved Brann to Magni's court. Previously he had no court assigned in the history file, and often ended up in Muradin's court in Northrend on later start dates. Additionally, Brann has been assigned as Magni's antiquarian, to reflect his prominent role in the Explorer's League and his association with Archaeology.

## Changelog:
- Set Magni as Brann's employer once he becomes King of Ironforge. (Wiki states he fought in the Second War during the Siege of Ironforge, so this seems accurate)
- Has Magni appoint Brann as Antiquarian

## Tests:

# How to test:
1. Begin a game in any start date.
2. View Magni's court and verify Brann is present and employed as an Antiquarian.
